lib/galaxy/datatypes/binary.py

```diff
@@ -3759,6 +3759,34 @@ def sniff(self, filename: str) -> bool:
         return Fast5Archive.sniff(self, filename)
 
 
+class Pod5(Binary):
+    """
+    Class describing a POD5 file. The POD5 Format Specification is at
+    https://pod5-file-format.readthedocs.io/en/latest/SPECIFICATION.html
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('test.pod5')
+    >>> Pod5().sniff(fname)
+    True
+    >>> fname = get_test_fname('test.fast5.tar')
+    >>> Pod5().sniff(fname)
+    False
+    """
+
+    file_ext = "pod5"
+
+    def sniff(self, filename: str) -> bool:
+        expected_signature = bytes([0x8B, 0x50, 0x4F, 0x44, 0x0D, 0x0A, 0x1A, 0x0A])
+        try:
+            with open(filename, "rb") as f:
+                first_8_bytes = f.read(8)
+                f.seek(-8, 2)
+                last_8_bytes = f.read(8)
+                return first_8_bytes == expected_signature and last_8_bytes == expected_signature
+        except Exception:
+            return False
```

Comment:
You can simplify the code by removing the try/except as by default Galaxy consider as "false" an unhandled exception in a sniffer.

Reply:
Thanks, I've removed that.